src/main/features/claimant-response/index.ts

```diff
@@ -44,7 +44,11 @@ export class ClaimantResponseFeature {
     app.all(/^\/case\/.+\/claimant-response\/(?!confirmation).*$/,
       DraftMiddleware.requestHandler(new DraftService(), 'claimantResponse', 100, (value: any): DraftClaimantResponse => {
         return new DraftClaimantResponse().deserialize(value)
-      }))
+      }),
+      (req: express.Request, res: express.Response, next: express.NextFunction) => {
+        res.locals.draft = res.locals.claimantResponseDraft
```

Comment:
Alias so that we do not have to specify what draft we deal with in components.

In the future we could change middleware to populate draft local env everywhere as we always have one draft per journey.